resources/src/main/java/org/cdshooks/CoverageRequirements.java

```diff
@@ -68,6 +70,24 @@ public String getQuestionnaireFaceToFaceUri() {
     return this.questionnaireFaceToFaceUri;
   }
 
+  public CoverageRequirements setQuestionnairePrescriberEnrollmenteUri(String questionnaireProviderEnrollmentUri) {
+    this.questionnaireProviderEnrollmentUri = questionnaireProviderEnrollmentUri;
+    return this;
+  }
+
+  public String getQuestionnaireProviderEnrollmentUri() {
+    return this.questionnaireProviderEnrollmentUri;
+  }
+
+  public CoverageRequirements setQuestionnaireProviderKnowledgeAssessmentUri(String questionnaireProviderKnowledgeAssessmentUri) {
+    this.questionnaireProviderKnowledgeAssessmentUri = questionnaireProviderKnowledgeAssessmentUri;
+    return this;
+  }
+
+  public String getQuestionnaireProviderKnowledgeAssessmentUri() {
+    return this.questionnaireProviderKnowledgeAssessmentUri;
```

Comment:
Here as well. These are all named `Provider` but I think should be `Prescriber`?